### PR TITLE
Implemented states and mock up for axios and local storage

### PR DESCRIPTION
**Pull request by jelicbogdan**
Need review and team talk about next steps

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "@types/node": "^16.18.3",
     "@types/react": "^18.0.25",
     "@types/react-dom": "^18.0.8",
+    "axios": "^1.2.0",
```
Comment (lanchemi11) [inline package.json:16]: Do we need this packet right now?

Comment (jelicbogdan) [reply package.json:16]: Not right now, but we need to discuss about fetching an API or using axios...

File: src/storage/LocalStorage.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  useEffect(() => {
+    try {
+      localStorage.setItem(key, JSON.stringify(value));
+    } catch (error) {
+      console.log(error);
+    }
+  }, [key, value]);
+
+  return [value, setValue];
+};
```
Comment (lanchemi11) [inline src/storage/LocalStorage.tsx:28]: Can you simplify this somehow?

Comment (jelicbogdan) [reply src/storage/LocalStorage.tsx:28]: Yes, working on it right now

**Pull request merged**